code/modules/buildmode/submodes/antirad_mode.dm

```diff
@@ -0,0 +1,49 @@
+/datum/buildmode_mode/clean_radiation
+	key = "rad"
+
+/datum/buildmode_mode/clean_radiation/show_help(mob/user)
+	to_chat(user, "<span class='notice'>***********************************************************</span>")
+	to_chat(user, "<span class='notice'>Left Mouse Button on obj/turf/mob = Remove radiation of selected target.</span>")
+	to_chat(user, "<span class='notice'>Ctrl-Left Mouse Button on screen = Remove all radiation in your sight.</span>")
+	to_chat(user, "<span class='notice'>Right Mouse Button on obj/turf/mob = Deep clean: Remove radiation of all recursive contents of turf.</span>")
+	to_chat(user, "<span class='notice'>Ctrl-Right Mouse Button on screen = Deep clean: Remove radiation of all recursive contents in your sight.</span>")
+	to_chat(user, "<span class='notice'>***********************************************************</span>")
```

Comment:
Can you make these into a list and then `.Join()` them?

Reply:
I was considering that, but then I feel like I would need to refactor ALL of them to use chat boxes and message lists, which I don't really want to do for a PR of this size